src/frontend/js/login.js

```diff
@@ -149,11 +217,38 @@ div:has(#submit_btn) {
     this.password = inputEl.value;
   }
 
+  checkLogin(e) {
+    if (e.key == "Enter") this.loginConfirm();
+  }
+
   setVisibility() {
-    this.pswVisibility = !this.pswVisibility
+    this.pswVisibility = !this.pswVisibility;
+  }
+
+  getElement(query) {
+    return this.renderRoot.querySelector(query) ?? null;
   }
 
   loginConfirm() {
+    if (
+      this.getElement("#username").value === "" &&
+      this.getElement("#password").value === ""
+    ) {
+      this.getElement("#usernameError").innerText = "*Inserisci uno username";
+      this.getElement("#passwordError").innerText = "*Inserisci una password";
```

Comment:
stessa cosa del commento precedente così come nelle riche successive andrebbe sistemato tutto in conseguenza delle modifiche suggerite

Reply:
modificate